### DISCUSSION TITLE
Laravel 11 and Filament 3 localhost:8000

### DISCUSSION THREAD

**Discussion by princeofforest** [filamentphp/filament#13655] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hi there,
Thank you for helping me. I have an issue when on editing page, the uploaded image is keep spinning, it cause of CORS block the image loading, however, it show on the list page. i have been search all over the internet, but have no luck. ChatGPT suggest to create middleware, still not working. I am using MAC MAMP for database, laravel serve. 
<img width="1710" alt="Screenshot 2024-07-22 at 7 50 20 AM" src="https://github.com/user-attachments/assets/05a03524-cb4d-438b-a01f-3e790f31278e">

Once again, thanks a lot for taking time to read this thread

**Comment by krekas** [comment]
Read about cors. You haven't set correctly `APP_URL`. It is different from what you trying to access

**Comment by leandrocfe** [comment]
use `http://localhost:8000` in your browser and set `APP_URL` in your .env file to the same address. Also, remember to run `php artisan storage:link`
